rts/lra/run.sh

```diff
@@ -59,32 +55,20 @@ function getDebugArgs {
 
 cd "$( dirname "${BASH_SOURCE[0]}" )"
 
-case "$(uname)" in
-   CYGWIN*) export NARAYANA_INSTALL_LOCATION=`cygpath -w $(pwd)/narayana-full-7.0.3.Final-SNAPSHOT` ;;
-   *)       export NARAYANA_INSTALL_LOCATION=$(pwd)/narayana-full-7.0.3.Final-SNAPSHOT ;;
-esac
-
-rm -rf $NARAYANA_INSTALL_LOCATION
-NARAYANA_ZIP="narayana-full-7.0.3.Final-SNAPSHOT-bin.zip"
-[ ! -f "$WORKSPACE/$NARAYANA_ZIP" ] &&\
-   echo "There is no Narayana zip at \$WORKSPACE directory at '$WORKSPACE/$NARAYANA_ZIP" && exit 1
-unzip "$WORKSPACE/$NARAYANA_ZIP"
-
 CURL_IP_OPTS=""
 IP_OPTS="${IPV6_OPTS}" # use setup of IPv6 if it's defined, otherwise go with IPv4
 if [ -z "$IP_OPTS" ]; then
   IP_OPTS="-Djava.net.preferIPv4Stack=true -Djava.net.preferIPv4Addresses"
   CURL_IP_OPTS="-4"
 fi
 
-echo "Narayana installed location = $NARAYANA_INSTALL_LOCATION"
-  java ${IP_OPTS} -Dquarkus.http.port=8080 $(getDebugArgs $PORT) -jar $NARAYANA_INSTALL_LOCATION/rts/lra/lra-coordinator-runner.jar &
+  java ${IP_OPTS} -Dquarkus.http.port=8080 $(getDebugArgs $PORT) -jar $WORKSPACE/rts/lra-examples/coordinator-quarkus/target/lra-coordinator-quarkus-runner.jar &
```

Comment:
This quickstart depends on the lra-examples quickstart being built first but there isn't a mention of that and it was not obvious to me. Please can I ask you to try to surface that somewhere? And also if `$WORKSPACE/rts/lra-examples/coordinator-quarkus/target/lra-coordinator-quarkus-runner.jar` does not exist please can I ask you to consider to make the script fail at that point?

Reply:
Thanks, I have updated the script. Do you think I should also mention it in the README?

Reply:
Yes, I think it would be appropriate to mention something there. How about mentioning to make sure to have built $WORKSPACE/rts/lra-examples in the `Build the demo` section?

Reply:
I put in in the 'Before you start' section, I think that is the appropriate place.

Reply:
Thank you!